### PR TITLE
Ignore dependabot and pre-commit-ci PR for release

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

PR of @dependabot and @pre-commit-ci is noisy to read [v0.43.0 release note](https://github.com/pyvista/pyvista/releases/tag/v0.43.0). Let's add [ignore-for-release](https://github.com/pyvista/pyvista/labels/ignore-for-release) to them and remove it from the release note.

### Details

- None

### CODE REVIEW

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
       with:
         labels: maintenance
     - uses: actions-ecosystem/action-add-labels@v1
-      if: startsWith(github.event.pull_request.head.ref, 'junk')
+      if: startsWith(github.event.pull_request.head.ref, 'junk') || startsWith(github.event.pull_request.head.ref, 'dependabot') || startsWith(github.event.pull_request.head.ref, 'pre-commit-ci-update-config')
```
Comment (banesullivan) [inline .github/workflows/label.yml:29]: I'm wondering if we should do this per user/app. For example ignoring the following users:

- `app/pre-commit-ci`
- `app/github-actions`
- `app/dependable`

Comment (tkoyama010) [reply .github/workflows/label.yml:29]: Yes. Let's!

Comment (tkoyama010) [reply .github/workflows/label.yml]: Seems that we can use `github.actor` for this.

**Review by banesullivan — APPROVED** [review]
LGTM, thanks @tkoyama010!

**Pull request merged**